probe-rs/src/debug/unit_info.rs

```diff
@@ -20,10 +20,10 @@ impl<'debuginfo> UnitInfo<'debuginfo> {
     /// If `find_inlined` is `false`, then the result will contain a single [`FunctionDie`]
     pub(crate) fn get_function_dies(
         &self,
-        address: u64,
+        address: RegisterValue,
```

Comment:
Why change the type here? To me an address is the extracted value, not a RegisterValue.  I think of cases like asking a debugger to decode an address to a symbol name - the input isn't always from a register like `pc`.

Reply:
It 'felt' cleaner to me at the taime. That said, you make a valid point, so perhaps one of the other reviewers will weigh in and sway the vote :)

Reply:
Just reconsidered after reading your comment in the inlined_function.rs test. What should the datatype for the address be? `u64`.

Reply:
Yes, u64.  I can think of exactly one ISA that has a 128-bit address space and it's never been actually produced.  Even on complex peripheral buses where you some times see a 96-bit physical address space it's mapped to a 64-bit virtual address space by the time it gets to the CPU / debugger.